tanner/emulators/cmd_exec.py

```diff
@@ -51,10 +53,14 @@ def __init__(self):
         return container
 
     async def get_cmd_exec_results(self, container, cmd):
-        execute_result = None
+        execute_result = ''
         try:
             container.start()
-            execute_result = container.exec_run(['sh', '-c', cmd]).decode('utf-8')
+            executor = ThreadPoolExecutor(max_workers=1)
+            cmd_future = executor.submit(container.exec_run, ['sh', '-c', cmd])
+            time.sleep(1)
+            if(cmd_future.done()):
+                execute_result = cmd_future.result().decode('utf-8')
```

Comment:
Maybe it is better to use timeout instead of blocking sleep?
https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Future.result